app/assets/stylesheets/searchworks4/typography.css

```diff
@@ -77,3 +77,8 @@ aside,
 .text-poppy {
   color: var(--stanford-poppy-dark);
 }
+
+.btn-link {
```

Comment:
Should this be in the component library?

Reply:
I don't know. I assume the component library (intentionally?) broke bootstrap's defaults and have no interest in going down that rabbit hole.

Reply:
It set the values based on the specifications given to us by DLUX.  I think it makes sense to have a single correct style for `.btn-link`.  If it's not correct in the component library, we ought to get it fixed.